config/opensearch_dashboards.yml

```diff
@@ -384,4 +384,7 @@
 # banner.enabled: true
 
 # Set the content of the banner
-# banner.content: "This is an important announcement for all users. [Learn more](https://opensearch.org) about OpenSearch."
\ No newline at end of file
+# banner.content: "This is an important announcement for all users. [Learn more](https://opensearch.org) about OpenSearch."
+
+# Set the value to true to enable the keyboard shortcuts
+opensearchDashboards.keyboardShortcuts.enabled: false
```

Comment:
lets comment this out as default

Reply:
+1.

Also we can modify the start:explore script to pass in this config set to true (within the package.json) so devs will have this on by default